lib/iris/tests/test_interpolation.py

```diff
@@ -510,25 +510,6 @@ def test_points_datatype_casting(self):
         # this test tries to extract a float from an array of type integer. the result should be of type float.
         r = iris.analysis.interpolate.linear(self.simple2d_cube_extended, [('shared_x_coord', 7.5)])
         self.assertCML(r, ('analysis', 'interpolation', 'linear', 'simple_casting_datatype.cml'))
-        
```

Comment:
These tests exercise the exact codepaths that previously failed, as fixed in #155.
These cases now have new tests:
 * 'test_mask' exercises the case where -- before the original fix, and in earlier versions of numpy -- it would return an array whose mask was a different shape than the data. 
   * now replaced by ```test_circular_src_masked``` [here](https://github.com/SciTools/iris/pull/1963/files#diff-b79409258b6aa9758c83b4f54c8d09d0R1245)
 * 'test_scalar_mask' checks the case mask=False, which was bugged in earliest version of the fix
    * now replaced by ```test_circular_src__masked_missingmask``` [here](https://github.com/SciTools/iris/pull/1963/files#diff-b79409258b6aa9758c83b4f54c8d09d0R1233)